src/test/java/underdogs/devbie/answer/service/AnswerServiceTest.java

```diff
@@ -0,0 +1,270 @@
+            .id(USER_ID)
+            .oauthId(TEST_OAUTH_ID)
+            .email(TEST_USER_EMAIL)
+            .build();
+        Answer expected = Answer.builder()
+            .id(1L)
+            .userId(USER_ID)
+            .questionId(QUESTION_ID)
+            .content(AnswerContent.from(TEST_ANSWER_CONTENT))
+            .build();
+        AnswerCreateRequest answerCreateRequest = new AnswerCreateRequest(QUESTION_ID, TEST_ANSWER_CONTENT);
+        given(answerRepository.save(any(Answer.class))).willReturn(expected);
+
+        Long id = answerService.save(user, answerCreateRequest);
+
+        assertThat(id).isEqualTo(expected.getId());
+    }
+
+    @DisplayName("면접 답변 전체 조회")
+    @Test
+    void readAll() {
+        Answer expectAnswer = Answer.builder()
+            .id(1L)
+            .userId(2L)
+            .questionId(3L)
+            .content(AnswerContent.from(TEST_ANSWER_CONTENT))
+            .build();
+        given(answerRepository.findAll()).willReturn(Collections.singletonList(expectAnswer));
+
+        AnswerResponses answerResponses = answerService.readAll();
+
+        assertThat(answerResponses).isNotNull();
+        assertThat(answerResponses.getAnswerResponses()).isNotNull();
+        List<AnswerResponse> actual = answerResponses.getAnswerResponses();
+        assertAll(
+            () -> assertThat(actual.get(0).getId()).isEqualTo(1L),
+            () -> assertThat(actual.get(0).getUserId()).isEqualTo(2L),
+            () -> assertThat(actual.get(0).getQuestionId()).isEqualTo(3L),
+            () -> assertThat(actual.get(0).getContent()).isEqualTo(TEST_ANSWER_CONTENT)
+        );
+    }
+
+    @DisplayName("하나의 면접 답 조회")
+    @Test
+    void read() throws Exception {
+        Answer expectAnswer = Answer.builder()
+            .id(1L)
+            .userId(2L)
+            .questionId(3L)
+            .content(AnswerContent.from(TEST_ANSWER_CONTENT))
+            .build();
+        given(answerRepository.findById(anyLong()))
+            .willReturn(Optional.of(expectAnswer));
+
+        AnswerResponse actual = answerService.read(1L);
+
+        assertAll(
+            () -> assertThat(actual).isNotNull(),
+            () -> assertEquals(actual.getId(), 1L),
+            () -> assertEquals(actual.getUserId(), 2L),
+            () -> assertEquals(actual.getQuestionId(), 3L),
+            () -> assertEquals(actual.getContent(), TEST_ANSWER_CONTENT)
+        );
+    }
+
+    @DisplayName("하나의 면접 답 조회 실패 - 존재하지 않는 Answer")
+    @Test
+    void readFailCauseByNotExistedAnswer() throws Exception {
+        Answer expectAnswer = Answer.builder()
+            .id(1L)
+            .userId(2L)
+            .questionId(3L)
+            .content(AnswerContent.from(TEST_ANSWER_CONTENT))
+            .build();
+        given(answerRepository.findById(anyLong())).willThrow(AnswerNotExistedException.class);
+
+        assertThatThrownBy(
+            () -> {
+                answerService.read(1L);
+            }
+        );
```

Comment:
```suggestion
        assertThatThrownBy(() -> answerService.read(1L));
```
람다식에 코드가 한줄이면 {} 중괄호를 안쓰기로 한 것 같습니다!